uint8 n_robot
int64 epoch
int16 pos_x
int16 pos_y
int16 theta
uint8 status_bola
int16 bola_x
int16 bola_y
int16 robot_condition
uint8 target_umpan
uint8 index_point
uint8 obs_length
int16[] obs_dist
uint8[] obs_index
float32 battery_health
int16 pos_x_odom
int16 pos_y_odom
int16 pos_theta_odom
int16 vx_icp
int16 vy_icp
int16 ball_next_x
int16 ball_next_y
int16 robot_next_x
int16 robot_next_y
int16 goalkeeper_field_x
int16 goalkeeper_field_y